stripe_mock_server/resources.py

```diff
@@ -854,10 +856,8 @@ def _set_up_subscription_and_invoice(self, plan):
         elif plan.interval == 'week':
             current_period_end += timedelta(days=7)
         elif plan.interval == 'month':
-            current_period_end = current_period_start.replace(
-                month=(current_period_start.month % 12) + 1,
-                year=current_period_start.year +
-                int(current_period_start.month / 12))
+            delta = relativedelta(months=1)
+            current_period_end = current_period_start + delta
```

Comment:
While you're at it, can you make it looks nice and consistent for months and years?
```py
        if plan.interval == 'day':
            current_period_end += timedelta(days=1)
        elif plan.interval == 'week':
            current_period_end += timedelta(days=7)
        elif plan.interval == 'month':
            current_period_end += relativedelta(months=1)
        elif plan.interval == 'year':
            current_period_end += relativedelta(years=1)
```